sequelize/sequelize-tests.ts

```diff
@@ -213,4 +213,215 @@ promiseMe = myModelInst.increment({});
 promiseMe = myModelInst.decrement({}, incrOpts);
 isBool = myModelInst.equal(myModelInst);
 isBool = myModelInst.equalsOneOf([myModelInst]);
-myModelPojo = myModelInst.toJSON();
\ No newline at end of file
+myModelPojo = myModelInst.toJSON();
+
+
+// data types test
+var types:any = Sequelize.STRING;
```

Comment:
Might be neater to have `var DataTypes: Sequelize.DataTypes = Sequelize;` and then reference the values on that?